Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14-alpine
+
+WORKDIR /excalidraw-room
+
+COPY package.json yarn.lock ./
+COPY scripts ./scripts
+RUN SKIP_YARN_POSTINSTALL=true yarn
```

Comment:
Copying in `package.json` and `yarn.lock` without the rest of the code and running `yarn` lets you cache Yarn installs.